nixnet/_session/frames.py

```diff
@@ -22,6 +22,25 @@ def __repr__(self):
     def _create_item(self, handle, index, name):
         return Frame(handle, index, name)
 
+    @property
+    def payld_len_max(self):
```

Comment:
I am not sure why it didn't run. The current Jenkinsfile in the master branch only clones the repo and doesn't actually perform any tests.